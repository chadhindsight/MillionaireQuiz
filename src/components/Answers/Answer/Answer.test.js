import React from 'react';
import { shallow } from 'enzyme';
import Answer from './Answer';

describe('Answers component', () =>{
    it('should render correctly with related child component info', ()=>{
        const component = shallow(<Answer />);

        expect(component).toMatchSnapshot();
    })
})

