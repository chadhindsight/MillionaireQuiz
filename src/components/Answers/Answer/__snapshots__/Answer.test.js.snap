// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Answers component should render correctly with related child component info 1`] = `
<div
  className="AnswerBox"
  style={
    Object {
      "visibility": "visible",
    }
  }
>
  <input
    type="checkbox"
  />
  <label
    className="correct"
  >
    <span>
      :
    </span>
    <p />
  </label>
</div>
`;
